tests/e2e/Makefile

```diff
@@ -40,4 +41,5 @@ e2e-test:
 		-log-artifacts=$(LOG_ARTIFACTS) \
 		-helm-chart=$(HELM_CHART) \
 		-ginkgo.focus="\[nvidia\]" \
-		-test.timeout=1h
+		-test.timeout=1h \
+		-collect-logs=$(COLLECT_LOGS_FROM)
```

Comment:
Does it make sense to optionally add this argument if `COLLECT_LOGS_FROM` is defined instead? `default` is already defined as the default value in the go code.

Reply:
I see no harm, since `COLLECT_LOGS_FROM` is set to `"default"` , we won't trigger anything special.

Reply:
My question was more around defining the default behaviour in two places -- meaning that we need to keep these in sync.

Reply:
I get it, yeah makes sense to not have the flag set in the makefile.